arrayAnalysis.js

```diff
@@ -0,0 +1,19 @@
+const arrayAnalysis = (numbers) => {
+  const average = numbers.reduce((acc, cr) => {
+    // eslint-disable-next-line no-param-reassign
+    acc += cr;
+    return acc;
+  }) / numbers.length;
+  const min = Math.min(...numbers);
+  const max = Math.max(...numbers);
+  const { length } = numbers;
```

Comment:
You should have this variable at the top and use it also for `average`.

Reply:
Good call